docs/class5/module11/lab3.rst

```diff
@@ -0,0 +1,70 @@
+Lab 11.3: Add Key to Regkey Pool
+-----------------------------------
```

Comment:
make the link matching the title, this will make cloud docs rst builder job failing.
e.g.
Lab 11.2: title
-------------

Reply:
same